### DISCUSSION TITLE
Change colour of the ModernSlider

### DISCUSSION THREAD

**Discussion by xavizardKnight** [PieroCastillo/Aura.UI#20] [Q&A] [answered]
Hello!
(Don't know if I should ask in Discussions or in Issues, I think this is not a bug so I'll ask it here)

I'm using ModernSlider to remplace standard Avalonia sliders in my applications.
By default, ModernSliders have the slidebar foreground black and the background blue, which I don't personally like it because it seems that the slide goes from right-to-left instead of left-to-right; so I want to reverse the colours, so the slider goes from left-to-right with the dark background and blue slide foreground.

Here's an image of a left-to-right ModernSlider in it's current state and a mockup of how I want it to look like:
![ModernSlide-Mockup](https://user-images.githubusercontent.com/35200790/131549749-d549b5bb-4c98-4713-978c-dc2dd2668ccd.png)

I tried to add the Background and Foreground attributes to the AXAML tag with hexadecimal colour codes, but it does nothing. I would want to ask how to achieve this colour change so it looks like the above image.

Here's the code of the ModernSlider currently (there are a lot of ticks lol):
```
<ModernSlider IsSnapToTickEnabled="True" Name="lenslide" VerticalAlignment="Center" Height="35"
                                  IsDirectionReversed="False" Background="#151515" Foreground="#3D729B"
                                  Value="1" Minimum="{Binding lenSlideMin}"
                                  Maximum="{Binding lenSlideMax}" Width="170"
                                  PointerLeave="Lenslide_OnPointerLeave"
                                 Ticks="1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40"/>
```                                  
Using Ubuntu MATE 20.04 LTS, Avalonia 0.10.7 and Aura.UI 0.1.3.1.

**Comment by PieroCastillo** [comment, marked as answer]
so easy, just override the follow resources:

https://github.com/PieroCastillo/Aura.UI/blob/8807d6f6e16c4995497eb4d1e69b0d7537e1c436/src/Aura.UI.FluentTheme/AuraUIResources.Fluent.axaml#L83-L105

also, don't worry, the current behavior is the expected

**Reply by xavizardKnight** [reply]
Thanks for your tip! I added these resources in my App.axaml, and I swapped the TintColor and the FallbackColor from the increase tags to the decrease tags and vice versa.

Now the sliders behave as I wanted. Thank you for your help!
